tools/tsp-client/src/network.ts

```diff
@@ -10,6 +10,8 @@ export function isValidUrl(url: string) {
 }
 
 // Checks if a file exists locally
-export function doesFileExist(path: string): Promise<boolean> {
-  return stat(path).then(() => true).catch(() => false);
+export async function doesFileExist(path: string): Promise<boolean> {
+  return stat(path)
+    .then(() => true)
+    .catch(() => false);
```

Comment:
Here you could use async/await. It's more code but more modern JS

```ts
try {
   await stat(path)
   return true;
} catch {
   return false
}
```

Reply:
Using await in the latest commit on this branch